static/js/src/advantage/subscribe/react/components/Form/Feature/Feature.tsx

```diff
@@ -25,186 +25,313 @@ const Feature = () => {
-              </span>
-            </label>
+            <p>
+              Not sure? Run <code> pro security-status</code> to find out which
+              packages and repositories you are currently using.
+            </p>
+            <RadioInput
+              label="Pro - all repositories"
+              checked={feature === Features.pro}
+              value={Features.pro}
+              onChange={handleChange}
+              disabled={proDisabled}
+              data-testid="pro"
+            />
+            <hr />
+            <RadioInput
+              label="Infra only - limited subset"
+              checked={feature === Features.infra}
+              value={Features.infra}
+              onChange={handleChange}
+              disabled={infraOnlyDisabled}
+              data-testid="infra-only"
+            />
           </div>
         </Col>
+        <Col size={11} emptyMedium={2} emptyLarge={2}>
+          <hr className="is-fixed-width" />
+        </Col>
+      </Row>
+      <Row>
+        <Col size={2} emptyMedium={2} emptyLarge={2}>
+          <h1>{feature === Features.pro ? "25000" : "2300"}</h1>
+        </Col>
         <Col size={3}>
+          <p>
+            Covers {feature === Features.pro ? "25000" : "2300"} packages in
+            Ubuntu Main {feature === Features.pro ? "and Universe" : ""}
+          </p>
+        </Col>
+        <Col size={6}>
+          <div className="p-graphic-main" />
           <div
             className={classNames({
-              "p-card--radio--column": true,
-              "is-selected": feature === Features.infra,
+              "p-graphic-universe": feature === Features.pro,
+              "p-graphic-universe--empty": feature === Features.infra,
             })}
-          >
-            <label className="p-radio u-align-text--center">
-              <input
-                data-testid="infra-only"
-                className="p-radio__input"
-                autoComplete="off"
-                type="radio"
-                aria-labelledby={`infra-label`}
-                value={Features.infra}
-                checked={feature === Features.infra}
-                onChange={handleChange}
-                disabled={infraOnlyDisabled}
-              />
-              <span className="p-radio__label" id={`infra-label`}>
-                <RadioInput
-                  labelClassName="inner-label"
-                  label={"Ubuntu Pro (Infra-only)"}
-                  checked={feature === Features.infra}
-                  value={Features.infra}
-                  onChange={handleChange}
-                  disabled={infraOnlyDisabled}
+          />
+          <p className="p-text--small graphic-legend-main">
+            LTS standard security maintenance for Ubuntu Main (initial 5 years)
+          </p>
+          {feature === Features.pro && (
+            <p className="p-text--small graphic-legend-universe">
+              LTS Expanded Security Maintenance (ESM) for Ubuntu Main
```

Comment:
Change to `10 years of security maintenance for packages in Ubuntu Universe repository`